README.md

```diff
@@ -1,2 +1,16 @@
-# HW01
-Edit the readme
+# HW01 
+## First assignment
+### 07/06/20
+#### *Monty* **Kwan**
+##### New immunology student at UChicago
+
+Places I've lived
+
+
+* New York City
+* Cape Elizabeth, Maine
+* Poughkeepsie, New York
+
+I am a research tech at [Weill Cornell Medicine](https://weill.cornell.edu/)
+
+[Github photo](www.github.com/kwanm.png)
```

Comment:
Hey Monty, your code looks great, but for some reason it seems the link to this photo is not working!